net/overture/Makefile

```diff
@@ -18,6 +18,8 @@ PKG_USE_MIPS16:=0
 
 GO_PKG:=github.com/shawn1m/overture
 GO_PKG_BUILD_PKG:=$(GO_PKG)/main
+GO_PKG_LDFLAGS:=-s -w
```

Comment:
Stripping is automatically handled by the build system so these flags are not necessary.